package.json

```diff
@@ -152,6 +152,17 @@
     "text-encoding": "^0.7.0",
     "ts-loader": "^7.0.5"
   },
+  "resolutions": {
```

Comment:
Is there a means/mechanism for notifying or reminding us when it's time to remove these resolutions?

Reply:
I don't think we have an automated way to do that, so it'll be on us to remember to do so.